contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -29,11 +38,20 @@ func init() {
 }
 
 const (
-	tagAWSAgent     = "aws.agent"
-	tagAWSService   = "aws.service"
-	tagAWSOperation = "aws.operation"
-	tagAWSRegion    = "aws.region"
-	tagAWSRequestID = "aws.request_id"
+	tagAWSAgent           = "aws.agent"
+	tagAWSService         = "aws.service"
+	tagTopLevelAWSService = "aws_service"
```

Comment:
```suggestion
	tagService = "aws_service"
```
Could this just be called service instead, with a comment to the right describing precisely what "service" means in this context? It's pretty verbose as it is now.

Same comment below for `tagTopLevelRegion` -> `tagRegion`